neutrinoapp/src/Search.jsx

```diff
@@ -156,19 +162,21 @@ export default function Search () {
                     </Link>{}
                   </td>
                 </tr>
-              )
+              ) : message
             }
           </tbody>
         </table>
         <div className={classes.root}>
+        { !error400 ? displayWiki.length < 1 ? <p>No data found</p> : 
```

Comment:
Same suggestion as above

Reply:
I was keeping the non error state first but I have rearranged it.

Reply:
Thanks -- I see the sense of that, but now that you've changed it I guess just as easy to keep as it is.